schema/schema.sql

```diff
@@ -0,0 +1,7 @@
+-- Create the Game table that inherits from Item
+CREATE TABLE game (
```

Comment:
The requirement for database says that "As you cannot implement inheritance in the database tables - add all properties and associations from the parent Item class as table column", I kindly recommend to add all the properties in Item to the Game class.